scripts/cwagent/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN apt-get update &&  \
     apt-get install -y ca-certificates curl && \
     rm -rf /var/lib/apt/lists/*
 
-RUN curl -O https://amazon-cloud-watch-agent.s3.amazonaws.com/debian/amd64/1.237768.0/amazon-cloudwatch-agent.deb && \
+RUN curl -O https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb && \
```

Comment:
Are we sure we want to use the latest version? Is it possible to pin a major version at least?

Reply:
Good point. Will change that.